modules/dreamview/frontend/proto_bundle/sim_world_proto_bundle.json

```diff
@@ -268,7 +268,7 @@
                   "id": 27
                 },
                 "batteryPercentage": {
-                  "type": "uint32",
+                  "type": "int32",
```

Comment:
Do not need these changes.please checkout

Reply:
Done.